### PR TITLE
added my getReply function file

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

File: javascript/javascript1/week4/task_1.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
+  if (command.includes("hello my name is")) {
+    myName = array[4];
+    response = `nice to see you ${myName}`;
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:9]: You should also consider if the user sends 'hello' twice with the same name as the task asks. Print something else then.

File: javascript/javascript1/week4/task_1.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  } else if (command.includes("what is my name")) {
+    response = `my name is ${myName}`;
+  } else if (command.includes("add fishing")) {
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:12]: You are adding the task fx. `fishing` in a static way. You could rather compare with only `add ` this text and add the task whatever comes after `add `, right?

File: javascript/javascript1/week4/task_1.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    todo.push("fishing");
+    response = "added fishing to todo";
+  } else if (command.includes("add singing")) {
+    todo.push("singing");
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:16]: See, there could be 100 same commands, even some unknown tasks like `add travelling to new places` then it won't work, right?

File: javascript/javascript1/week4/task_1.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    response = "added singing to todo";
+  } else if (command.includes("remove fishing")) {
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:18]: Same thing applies for removing as well.

File: javascript/javascript1/week4/task_1.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December"
+    ];
+    const curMonth = months[today.getMonth()];
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:49]: It seems ok for now. But try to replace names like `curmonth` with `currentMonth`.

File: javascript/javascript1/week4/task_1.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    if (operator === "*") {
+      response = a * b;
+    }
+
+    if (operator === "-") {
+      response = a - b;
+    }
+  } else if (command.includes("set a timer")) {
+    function myFunction() {
+      console.log("timer set for 5 seconds");
```
Comment (TITHI1244) [inline javascript/javascript1/week4/task_1.js:70]: You cannot predict how much time the user will set. So keep it dynamic.

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
Nice work, but try to think of any inputs as variables and use them as dynamic values as much as you can :)

File: javascript/javascript1/week4/task_1.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
```
Comment (esteficodes) [inline javascript/javascript1/week4/task_1.js:6]: Hi Parul! Nice job with creating a variable for the response! (Or you can also just use return) 😊

File: javascript/javascript1/week4/task_1.js [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+      "September",
+      "October",
+      "November",
+      "December"
+    ];
+    const curMonth = months[today.getMonth()];
+    const curYear = today.getFullYear();
+    const dateOfMonth = today.getDate();
+    response = `today is ${dayOfWeek} , ${dateOfMonth} of ${curMonth} , ${curYear}`;
+  } else if (command.includes("What is ")) {
```
Comment (esteficodes) [inline javascript/javascript1/week4/task_1.js:53]: Good job with the conditional! You can also add an "else" at the end to catch any other possible outcomes as errors or wrong user inputs. 🤩

File: javascript/javascript1/week4/task_1.js [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    }
+  } else if (command.includes("set a timer")) {
+    function myFunction() {
+      console.log("timer set for 5 seconds");
+    }
+    setTimeout(myFunction, 5000); //sets time in milliseconds(5 seconds*1000)
+  }
+
+  return response;
+}
```
Comment (esteficodes) [inline javascript/javascript1/week4/task_1.js:76]: Nice timer function! 😎

**Review by esteficodes — COMMENTED** [review]
Well done programming the Voice assistant! You might want to comment your code with // so that you can easily see what you did here at one glance (and also to practice) 🤗

**Pull request merged**